containers/Reports/ReportCreateMessage.js

```diff
@@ -52,68 +52,66 @@ export const ReportCreateMessage = props => {
   };
 
   return (
-    <KeyboardAvoidingView
-      style={styles.descripContainer}
-      behavior={(Platform.OS === 'ios')? 'padding' : null}
-    >
-      <View style={styles.actionsContainer}>
-        <Button
-          buttonStyle={styles.backButton}
-          titleStyle={styles.backButtonText}
-          title='Back'
-          type='clear'
-          icon={arrowLeftIcon}
-          onPress={() => props.navigation.goBack()}
-        />
-      </View>
-      <TextSourceSans style={styles.title}>
-        You want to report this content? Your message will be sent to the
-        moderation. The moderation will look at the reported content. The
-        content will be deleted if it does not meet our discussion rules
-        (netiquette).
-      </TextSourceSans>
-      <Formik
-        validationSchema={reportMessageValidationSchema}
-        initialValues={{message: ''}}
-        onSubmit={values => handleSubmit(values)}
-        validateOnMount={true}
-      >
-        {({
-          handleChange,
-          handleBlur,
-          handleSubmit,
-          values,
-          errors,
-          touched,
-          isValid,
-        }) => (
-          <>
-            <TextInputFullFormField
-              field='Your Message'
-              name='message'
-              value={values.message}
-              placeholder='Add message'
-              returnKeyType='next'
-              returnKeyLabel='next'
-              onChangeText={handleChange('message')}
-              onBlur={handleBlur('message')}
-              error={errors.message}
-              touched={touched.message}
-              numberOfLines={10}
-              multiline={true}
+    <View style={styles.container}>
```

Comment:
the keyboard open/hide seems broken for Idea description and Report Modal.

android, google pixel 5
- clicking opens the keyboard, but overlaps the submit button (its possible to scroll though).
- hiding the keyboard again, the button is in the middle of the screen instead of the bottom of the screen

Reply:
@khamui the keyboard overlapping submit is expected or on small screens you can only see a line or so of what you are typing. 
The submit button in middle is new, will test on pixel 5 and see if can replicate.

Reply:
@khamui ok when fixing button position the overlapping also got fixed for smaller screen, hopefully